tensorflow/lite/g3doc/guide/build_cmake.md

```diff
@@ -292,8 +292,15 @@ cmake --build . -j
 
 This command generates the following shared library in the current directory.
 
+### Windows Only, For Release Build
+
 **Note:** On Windows system, you can find the `tensorflowlite_c.dll` under
-`debug` directory.
+`debug` directory by default.
+
+```sh
+cmake ../tensorflow_src/tensorflow/lite/c
+cmake --build . -j --config Release
+```
 
```

Comment:
I prefer to move the Windows section to the bottom or add new section title like "### Generated Library" here.